athena_udf/__init__.py

```diff
@@ -35,9 +42,25 @@ def handle_udf_request(cls, event):
         record_batch_list = record_batch.to_pylist()
 
         outputs = []
-        for record in record_batch_list:
-            output = cls.handle_athena_record(input_schema, output_schema, list(record.values()))
-            outputs.append(output)
+        with ThreadPoolExecutor() as executor:
+            if self.chunk_size is None:
```

Comment:
could you add an example in example about using chuck_size vs not using it?

`lambda_handler = AthenaPythonUDF(chunck_size=5000).lambda_handler`